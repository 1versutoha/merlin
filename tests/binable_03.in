["find", "use", "bin_prot"]
["tell", "source", "module Test_binprot : Bin_prot.Binable.S2 = struct type ('a,'b) t = T of 'a * 'b with bin_io end"]
["tell", "end"]
["errors"]
