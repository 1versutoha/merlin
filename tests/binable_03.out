["return",true]
["return",null]
["return",{"line":1,"col":96}]
["return",[]]
